src/gui/AstroCalcDialog.cpp

```diff
@@ -2738,7 +2735,7 @@ LunarEclipseIteration::LunarEclipseIteration(double &JD, double &positionAngle,
 void AstroCalcDialog::selectCurrentLunarEclipseDate(const QModelIndex& modelIndex)
 {
 	double JDMid = modelIndex.sibling(modelIndex.row(), LunarEclipseDate).data(Qt::UserRole).toDouble();
-	if (objectMgr->findAndSelectI18n("Moon") || objectMgr->findAndSelect("Moon"))
```

Comment:
Why is the search by localized name even needed, if it's statically defined here to be the Moon?

Reply:
That's OK, the replacement doesn't invalidate my question.

Reply:
I don't find this code in active use any longer. Of course this is unnecessary.

Reply:
Oh, just noticed that it was later replaced by `objectMgr->findAndSelectI18n(name) || objectMgr->findAndSelect(name)`, which makes a bit more sense as a more general code.